native/cocos/ui/videoplayer/VideoPlayer.h

```diff
@@ -66,6 +66,12 @@ class VideoPlayer : public RefCounted {
     };
 
     VideoPlayer();
+    ~VideoPlayer();
```

Comment:
override needed?

Reply:
No, It should not be inherited from `RefCounted`. I will remove that.